test/e2e/prometheus_test.go

```diff
@@ -167,3 +167,35 @@ func TestPrometheusAlertmanagerAntiAffinity(t *testing.T) {
 		t.Fatal("Can not find pods: prometheus-k8s or alertmanager-main")
 	}
 }
+
+func TestPrometheusTerminationMessagePolicy(t *testing.T) {
+	pods, err := f.KubeClient.CoreV1().Pods(f.Ns).List(
+		metav1.ListOptions{FieldSelector: "status.phase=Running"})
+	if err != nil {
+		t.Fatal(err)
+	}
+	var (
+		testNs = "openshift-monitoring"
+		clOper = "cluster-monitoring-operator"
+		v      string
```

Comment:
This doesn't need to be initialized here.

Reply:
Fixed